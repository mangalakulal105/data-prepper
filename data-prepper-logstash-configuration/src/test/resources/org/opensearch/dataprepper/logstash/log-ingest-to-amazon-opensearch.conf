input {
    http {
    }
}
filter {
    grok {
        match => {"log" => "%{COMBINEDAPACHELOG}"}
    }
}
output {
    amazon_es {
        hosts => ["fakedomain.us-east-1.es.amazonaws.com"]
        region => "us-east-1"
        index => "my-index"
    }
}