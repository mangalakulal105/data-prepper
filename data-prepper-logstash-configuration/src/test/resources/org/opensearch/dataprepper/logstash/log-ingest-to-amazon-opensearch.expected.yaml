logstash-converted-pipeline:
  source:
    http:
      max_connection_count: 500
      request_timeout: 10000
  processor:
    - grok:
        match:
          log:
            - "%{COMBINEDAPACHELOG}"
  sink:
    - opensearch:
        aws_sigv4: true
        insecure: false
        hosts:
          - "fakedomain.us-east-1.es.amazonaws.com"
        aws_region: "us-east-1"
        index: "my-index"
